src/app/rundown-view/components/offsetable-piece/offsetable-piece.component.ts

```diff
@@ -33,6 +36,51 @@ export class OffsetablePieceComponent {
   @ViewChild('labelTextElement')
   public labelTextElement: ElementRef<HTMLSpanElement>
 
+  public media: Media | undefined
+
+  public isMediaPiece: boolean = false
+
+  private mediaSubscription?: Subscription
```

Comment:
Since `mediaSubscription` is optional you could use that as your `isMediaPiece`.
If the subscription is `undefined` it is not a media piece. If it is not undefined, then it is. 
One less variable to maintain :)